### ISSUE TITLE
jspm@0.17beta9 fails to build static when nodelibs-process is involved

### ISSUE THREAD

**Issue by perrin4869** [jspm/jspm-cli#1593]
Last case I tried: https://github.com/perrin4869/jspm0.17_buildstatic
Works awesome except for buildStatic. Thanks!

**Comment by guybedford** [comment]
When you say the static build fails here, do you mean that it throws an error while running the build? Can you share the stack trace?

**Comment by perrin4869** [comment]
Yup, here's the stack trace:

```
Unhandled rejection TypeError: No resolution found at all for condition github:jspm/nodelibs-process@0.2.0-alpha/#:./process.js.
    at /home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/systemjs-builder/lib/trace.js:640:15
    at Array.forEach (native)
    at /home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/systemjs-builder/lib/trace.js:633:6
    at tryCatcher (/home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/bluebird/js/release/promise.js:503:31)
    at Promise._settlePromise (/home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/bluebird/js/release/promise.js:560:18)
    at Promise._settlePromise0 (/home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/bluebird/js/release/promise.js:605:10)
    at Promise._settlePromises (/home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/bluebird/js/release/promise.js:684:18)
    at Async._drainQueue (/home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/bluebird/js/release/async.js:126:16)
    at Async._drainQueues (/home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/bluebird/js/release/async.js:136:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/home/perrin4869/projects/frontend/jspm/buildstatic/node_modules/bluebird/js/release/async.js:16:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)

```

**Comment by guybedford** [comment]
I can confirm this is working on master for me, so this may already be fixed pending release.

**Comment by guybedford** [comment]
Try the released 0.17.0-beta.10, and let me know if you still have any issues.
